backend/calc.py

```diff
@@ -40,23 +62,47 @@ def get_multiplier(self):
 
 
 class ProbabalisticStat(Stat):
-    def __init__(self, stat, value):
+    """
+    Derived from Stat class, used for stats that increase the chance of something happening such as critical hit and
+    direct hit.
+
+    p_factor: something
+    p_scalar: something else
+    """
+
+    # Class variable to convert stats
+    crit_convert: ClassVar[dict[Stats, tuple[int, int]]] = {
```

Comment:
mark a todo for converting this into a less cancerous type in a separate pr. i'll file a corresponding issue.

Reply:
i just fixed it instead